src/garage/tf/models/__init__.py

```diff
@@ -15,7 +16,7 @@
 
 __all__ = [
     'CNNModel', 'CNNModelWithMaxPooling', 'LSTMModel', 'Model',
-    'GaussianGRUModel', 'GaussianLSTMModel', 'GaussianMLPModel', 'GRUModel',
-    'MLPDuelingModel', 'MLPMergeModel', 'MLPModel', 'NormalizedInputMLPModel',
-    'Sequential'
+    'GaussianConvModel', 'GaussianGRUModel', 'GaussianLSTMModel',
```

Comment:
Can you take this opportunity to rename all of the *Conv* primitives to *CNN* instead? it's more consistent with there rest of the codebase.

Reply:
sure, will do that